com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -608,10 +634,12 @@ static void OnAdditionalProbesBakeCompleted()
                 probeVolume.OnBakeCompleted();
             }
 
-            UnityEditor.AssetDatabase.SaveAssets();
-            UnityEditor.AssetDatabase.Refresh();
+            AssetDatabase.SaveAssets();
+            AssetDatabase.Refresh();
             probeRefVolume.clearAssetsOnVolumeClear = false;
 
+            m_BakingBatch = null;
```

Comment:
Clearing this exposes the fact that we still occasionally have multiple callbacks registered for completion (the second one will encounter this null and bail out with a callstack)